tests/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 # Copyright (c) Open Enclave SDK contributors.
 # Licensed under the MIT License.
 
-include(oeedl_file)
```

Comment:
Can this cmake module be deleted from the repo? Or does it continue to serve some purpose?

Reply:
Oh, will do. I intended to remove it but I guess I forgot :smile: